VWRs edits (#1193)
Macrometacorp/docs: docs/photoniq/vwrs/03-configuration-and-setup/02-configure-waiting-rooms.md

---
sidebar_position: 20
title: Configuring Waiting Room Parameters
---

This page explains some parameters to configure when using the VWR using API endpoint properties. These include configurations when:

- Entering the waiting room
- In the waiting room
- Exiting the waiting room

## Entering the Waiting Room

- **Rate Limit:** Set the rate (maximum users per second) at which traffic should be diverted to the waiting room. (`rate_limit` property)

> - **Waiting Room Path:** The URL that sits in front of the waiting room (`domain_url` property)

Reviewer: `domain_url` was renamed to `waitingroom_url`. Additionally, this description is confusing, and the title and API field names are both wrong. I'd change it to:

```suggestion
- **Waiting Room URL:** The URL that you are protecting with a waiting room (`waitingroom_url` property)
```

if you can think of a better sentence to use here, please go for it

- **Queue Type:** Defines how requests should be removed from the waiting room, such as FIFO (first in, first out). (`queue_type` property)

## In the Waiting Room

- **Waiting Room HTML:** UI shown to users in the waiting room. (`waiting_room_path` property)

> - **Waiting Room Position and Time (in HTML):** Include any of the position or time in the waiting room UI. This is (Configured in the [Akamai EdgeWorker](01-configuring-edgeworkers.md)

Reviewer 2: ```suggestion
- **Waiting Room Position and Time (in HTML):** Include any of the position or time in the waiting room UI. This is Configured in the [Akamai EdgeWorker](01-configuring-edgeworkers.md)
```
